install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1082,12 +1082,14 @@ hubble:
       # false, the certs for Hubble server need to be provided by setting
       # appropriate values below.
       enabled: true
-      # -- Set the method to auto-generate certificates. Supported values:
+      # -- Sets the method to auto-generate certificates. Supported values:
+      # - cronJob:      This method uses a Kubernetes CronJob to generate any
+      #                 certificates not provided by the user at
+      #                 installation time.
+      # - certmanager:  This method uses cert-manager to generate & rotate
+      #                 certificates.
       # - helm:         This method uses Helm to generate all certificates.
-      # - cronJob:      This method uses a Kubernetes CronJob the generate any
-      #                 certificates not provided by the user at installation
-      #                 time.
-      # - certmanager:  This method use cert-manager to generate & rotate certificates.
+      #                 Deprecated and will be removed in Cilium v1.15.
```

Comment:
I guess this should be 1.16 now. The same below, and in the commit message.

Reply:
Good catch thank you!